templates/search/search_clients_daily_v8/query.sql

```diff
@@ -1,3 +1,14 @@
+-- Return the version of the search addon if it exists, null otherwise
+CREATE TEMP FUNCTION
+  get_search_addon_version(addon_list ANY TYPE) AS ((
+    SELECT
+      element.version
+    FROM
+      UNNEST(addon_list)
+    WHERE
+      element.addon_id = 'followonsearch@mozilla.com'
```

Comment:
this needs a limit clause to prevent query errors if someone sends us garbage
```suggestion
      element.addon_id = 'followonsearch@mozilla.com'
    LIMIT
      1
```

Reply:
alternately we could attempt to get the max or mode-last version